db/schema.rb

```diff
@@ -71,6 +71,12 @@
     t.uuid "claim_school_id"
     t.uuid "current_school_id"
     t.integer "employment_status"
+    t.boolean "biology_taught"
+    t.boolean "chemistry_taught"
+    t.boolean "computer_science_taught"
+    t.boolean "languages_taught"
+    t.boolean "physics_taught"
```

Comment:
Was losing the default values intentional?

Reply:
yeah, I don't believe they are necessary, and leaving it `null` by default means we can distinguish between a claim that hasn't answered the questions and one that has answered with "No".